src/icons/tasks/IconTaskApproval.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
```

Comment:
I'm under the impression we intend to style icons via CSS classes exclusively going forward. @tjgupta, can you take a quick look?

Reply:
Yep, I can work with @shahzadaziz to conform this to our two-toned icon style.